evaluator/perception_online_evaluator/include/perception_online_evaluator/metrics_calculator.hpp

```diff
@@ -95,15 +100,34 @@ class MetricsCalculator
    */
   void setPredictedObjects(const PredictedObjects & objects);
 
+  void updateObjectsCountMap(const PredictedObjects & objects, const tf2_ros::Buffer & tf_buffer);
+
   HistoryPathMap getHistoryPathMap() const { return history_path_map_; }
   ObjectDataMap getDebugObjectData() const { return debug_target_object_; }
+  DetectionCountMap getDetectionCountMap() const { return historical_detection_count_map_; }
 
 private:
   std::shared_ptr<Parameters> parameters_;
 
   // Store predicted objects information and calculation results
   ObjectMap object_map_;
   HistoryPathMap history_path_map_;
+  DetectionCountMap initializeDetectionCountMap()
+  {
+    return {
+      {0, 0},  // UNKNOWN
+      {1, 0},  // CAR
+      {2, 0},  // TRUCK
+      {3, 0},  // BUS
+      {4, 0},  // TRAILER
+      {5, 0},  // MOTORCYCLE
+      {6, 0},  // BICYCLE
+      {7, 0},  // PEDESTRIAN
+    };
+  }
```

Comment:
could you please use `ObjectClassification::UNKNOWN;` as key?

Reply:
thanks! changed in https://github.com/autowarefoundation/autoware.universe/pull/6761/commits/d1c6f044c21b403b16675058f70ed394ed677b70